packages/thirdweb/src/react/core/hooks/connection/ConnectButtonProps.ts

```diff
@@ -13,6 +14,45 @@ import type {
 } from "../../utils/defaultTokens.js";
 import type { SiweAuthOptions } from "../auth/useSiweAuth.js";
 
+export type Currency = {
+  address: string;
+  name: string;
+  symbol: string;
+  decimals: number;
+  icon?: string;
+};
+
+export type PaymentInfo = {
+  /**
+   * The chain to receive the payment on.
+   */
+  chain: Chain;
+  /**
+   * The address of the seller wallet to receive the payment on.
+   */
+  sellerAddress: string;
+  /**
+   * Optional ERC20 token to receive the payment on.
+   * If not provided, the native token will be used.
+   */
+  currency?: Currency;
```

Comment:
should we use the term `token` instead of `currency` here?

Reply:
hmmm yeah maybe that's more explicit